### PR TITLE
Add browser tests

### LINKED ISSUE #123: run tests in the browser using e.g. zuul
see https://github.com/Level/abstract-leveldown/pull/93 (similar setup to what `memdown` does)

### LINKED ISSUE #220: Replace const with var for IE10
See https://github.com/Level/abstract-leveldown/issues/123#issuecomment-388588760.

### PR DESCRIPTION
**Pull request by vweevers**
Closes #123 and closes #220. Ready for review, but marking as WIP because I've yet to activate the OSS Sauce Labs account.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 language: node_js
 
-node_js:
-  - 10
-  - 9
-  - 8
-  - 6
-  - 4
+matrix:
+  fast_finish: true
```
Comment (ralphtheninja) [inline .travis.yml]: What does this do?

Comment (vweevers) [reply .travis.yml]: If one job fails, the entire build fails. Instead of waiting for other jobs.

**Comment by vweevers** [comment]
Rebased on master and squashed.

Will merge once Sauce Labs activates the account (probably monday/tuesday)

**Comment by vweevers** [comment]
[Sauce Test Status](https://saucelabs.com/u/abstract-leveldown)

**Pull request merged**